content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -508,6 +508,18 @@ you through the steps you follow to apply a seccomp profile to a Pod or to one o
 its containers. That tutorial covers the supported mechanism for configuring seccomp in Kubernetes,
 based on setting `securityContext` within the Pod's `.spec`.
 
+### snapshot.storage.kubernetes.io/allowVolumeModeChange
+
+Example: `snapshot.storage.kubernetes.io/allowVolumeModeChange: true`
+
+Used on: VolumeSnapshotContent
+
+Value can either be `true` or `false`.
+This determines whether a user can modify the mode of the source volume when a `PVC` is being created from a `VolumeSnapshot`.
+
+See [Converting the volume mode of a Snapshot](/docs/concepts/storage/volume-snapshots/#convert-volume-mode) for more information.
+
```

Comment:
@sftim Does it make sense to only keep this brief section for labels-annotations-taints and refer to https://kubernetes-csi.github.io/docs/ for details?

Reply:
@xing-yang : From a personal standpoint, I'd err on the side of too much information rather than too less. 
In the spirit of what I just said, what we could potentially rewrite it as is suggested above.

Reply:
Which specific page on https://kubernetes-csi.github.io/docs/ would we link to?